.travis.yml

```diff
@@ -53,6 +53,7 @@ branches:
   only:
     - master
     - impl-EZP-24434-aa-core-2
+    - fix-EZP-24556-solr-customfield
```

Comment:
Removed in https://github.com/ezsystems/ezpublish-kernel/commit/bbc43859858b474071cf403edb762a58e348550b